pkg/collector/corechecks/containers/docker.go

```diff
@@ -200,7 +200,7 @@ func (d *DockerCheck) Run() error {
 			if c.Memory.SwapPresent == true {
 				sender.Gauge("docker.mem.swap", float64(c.Memory.Swap), "", tags)
 			}
-
+			
```

Comment:
Let's remove these whitespaces. Also pro tip: you can configure your editor to do that automatically on file save :)